front/cypress/e2e/proposal_posting_anonymous.cy.ts

```diff
@@ -0,0 +1,118 @@
+import moment = require('moment');
+import { randomString } from '../../../support/commands';
+
+describe('Initiatives with anonymous participation allowed', () => {
+  const projectTitle = randomString();
+  const projectDescriptionPreview = randomString(30);
+  let projectId: string;
+  let projectSlug: string;
+
+  before(() => {
+    cy.setAdminLoginCookie();
+    cy.visit('/admin/initiatives/settings');
+    cy.apiGetAppConfiguration().then((response) => {
+      if (
+        !response.body.data.attributes.settings.initiatives
+          .allow_anonymous_participation
+      ) {
+        cy.get('#e2e-anonymous-posting-toggle').should('exist');
+        cy.get('#e2e-anonymous-posting-toggle').click();
+        cy.get('#e2e-initiative-settings-submit-button').click();
+      }
+    });
+  });
+
+  it('admin can submit anonymous initiatives', () => {
+    const initiativeTitle = randomString(20);
+    const initiativeContent = randomString(60);
+
+    cy.setAdminLoginCookie();
+
+    cy.visit(`/initiatives/new`);
+    cy.get('#e2e-initiative-title-input').as('titleInput');
+    cy.get('#e2e-initiative-form-description-section .ql-editor').as(
+      'descriptionInput'
+    );
+    cy.contains('What is your proposal?').should('exist');
+
+    // add title and description
+    cy.get('@titleInput').type(initiativeTitle);
+    cy.get('@descriptionInput').type(initiativeContent, { delay: 1 });
+
+    // verify the values
+    cy.get('@titleInput').should('contain.value', initiativeTitle);
+    cy.get('@descriptionInput').contains(initiativeContent);
+
+    // add a topic
+    cy.get('.e2e-topics-picker').find('button').eq(3).click();
+
+    // set to anonymous
+    cy.get('#e2e-post-anonymously-checkbox').should('exist');
+    cy.get('#e2e-post-anonymously-checkbox').click();
+    cy.wait(1000);
```

Comment:
The `wait` here and after the modal pops up seems to be needed :thinking:  Otherwise the clicks happen too quickly and aren't registered - I'm guessing this has to do with the strange bounced API calls that the proposal form makes (what I struggled with last week). Adding a short wait solves the flakiness in this case though.